### PR TITLE
Issue #4144: Modified LocalFinalVariableNameCheckest.java and move its tests to subdir

### PR DESCRIPTION
**Pull request by kuri-leo**
//I did every step again to make sure that I followed the requirements in Issue #3891 and PR rules

this is in response of the Issue #4144

changes including:
1. move the test files of LocalFinalVariableName to subdir and rename them.
2. change code in LocalFinalVariableNameCheckest to adapt the path.

Results:

1. All unit tests passed.
2. the result of `mvn clean verify` -> `Tests run: 103, Failures: 0, Errors: 0, Skipped: 0`

I'm sorry for my closing of PR #4146 and #4147
For the first, I found my mistake and wanted to reduce your work, so I cloesd my pull request.
But for the second, I found that I named my branch incorrectly then I deleted it, but my request was closed automatically. 
I'm terriablly sorry for my violation of the PR rules and please forgive me.

### CODE REVIEW

**Comment by MEZk** [comment]
@kuri-leo 
You still do not follow PR rules.

> Follow the instructions and do it again

We do not need to know this. State what the commit does in the commit message.

**Comment by rnveach** [comment]
@kuri-leo The PR rules also stated to not create new PRs. You can make all modifications to the PR without closing it.

**Comment by kuri-leo** [comment]
@MEZk @rnveach 
**I'm sorry for my mistake**, I never use PR before, and my friends told me that I should close it and make new PR. 
I will practice how to use PR by creating new repository in my own github account.
But for this PR, should I reset my commit and make a  a good commit message instead?

**Comment by rnveach** [comment]
> But for this PR, should I reset my commit and make a a good commit message instead?

You can amend previous commits. You don't need to wipe it out and start over again.

**Comment by kuri-leo** [comment]
@rnveach thank you for your guide, I've edited my commit. and please check it. I promise this is the last time I make such a sally mistake.

**Comment by rnveach** [comment]
@kuri-leo Running `mvn verify` locally will guide you through most of your issues.
It will fail on your commit message and guide you to the correct way.

**Comment by kuri-leo** [comment]
@rnveach now all the tests has passed. I will add those experience to my notebook. and thanks for your guide again!

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import java.io.*;
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputSimple
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java]: Please rename class to `InputLocalFinalVariableName`.

Comment (kuri-leo) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/naming/localfinalvariablename/InputLocalFinalVariableName.java]: @rnveach my code has updated, please check! Thanks! ;)

**Pull request merged**